www/pages/index.md

```diff
@@ -69,6 +69,7 @@ Another option is to use CSS. The component has a `headroom` class as well as a
 
 *   `onPin` — callback called when header is pinned
 *   `onUnpin` — callback called when header is unpinned
+*   `onUnfix` — callback called when header position is no longer fixed
```

Comment:
"onUnfix" is not a correct name. It should be "onFix" because it fires when header position is fixed

Reply:
This PR simply added the already existing function by that name to the documentation. If you'd like to rename it, send a new PR for that